tests/search_intentions/test_field_characteristics.py

```diff
@@ -129,6 +129,18 @@ def any_words_in_string(include_words: list[str], string: str) -> bool:
         all_or_any="all",
         known_failure=False,
     ),
+    FieldCharacteristicsTestCase(
+        search_terms="mango",
+        test_field="text_block_text",
+        exact_match=False,
+        characteristics_test=(lambda x: x not in {"Avian Influenza", "Earthquake"}),
+        description="Semantic search threshold should apply.",
+        # slug: national-disaster-risk-management-plan-2015_3f8f
```

Comment:
could this comment be moved into the test description?

Reply:
good idea!